File: inf.c
#include<stdio.h>
#include<string.h>
int main()
{
char arr[100];
int t;
scanf("%d",&t);
while(t--)
{
int k=1;
scanf("%s",arr);
int l=strlen(arr);
int ars[26]={0};

int i;
for(i=0;i<l;i++)
{
ars[arr[i]-97]++;
}

for(i=0;i<26;i++)
{
if(ars[i]==0)
{
k=0;
}
}
if(k==0)
printf("NO\n");
else
printf("YES\n");
}
return 0;
}

File: Inf.java
import java.util.Scanner;
class Inf
{
public static void main(String args[])
{
Scanner st=new Scanner(System.in);
String s=st.nextLine();
int l=s.length();
int i;
String rev="";
char arr[]=s.toCharArray();
int visited[]=new int[26];
int ars[]=new int[26];
for(i=0;i<26;i++)
{
ars[i]=0;
}
for(i=0;i<l;i++)
{
ars[arr[i]-97]++;
}
for(i=0;i<26;i++)
{
visited[i]=0;
}
for(i=0;i<l;i++)
{
if(visited[arr[i]-97]==0)
{
rev=rev+arr[i];
visited[arr[i]-97]=1;
}
}
System.out.println(rev);
}
}